kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/controllers/IdentifierController.java

```diff
@@ -37,7 +37,9 @@ public Identifier createIdentifier(String type, String source, String sourceId)
    * @param source source of identifier
    * @param sourceId id in source system
    * @return created identifier
+   * @deprecated
    */
+  @Deprecated
   public Identifier createIdentifier(IdType type, String source, String sourceId) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1133)